### PR TITLE
[FEAT/OPT] i18n on tags; transform tag path to English

### PR DESCRIPTION
**Pull request by dextermallo**
# Initiates

I am a rookie for CTF and just found this fantastic tool recently, hope I can contribute a bit :)
(P.S. I only learned Vue a little around 3 years ago, please do let me know if there's anything I can improve)

# Changes

## Optimization
1. Support i18n on function `tags`: It would be great if all the text displayed the exact language people choose :). Noted that it only supports those tools with `displayed: true`.

## Feature
2. Transform the tag path to English-only: When clicking on the tag (`https://ctfever.uniiem.com/tag/<tagType>`) it will display with encoded one right now (eg. https://ctfever.uniiem.com/tag/%E5%AD%97%E7%AC%A6%E7%BC%96%E7%A0%81). It would be prettier when using English-only just like the other url.

## Chore
3. Tag changes: Since there's new support for i18n on tags, as a person who just joins into the CTF recently, please kindly help to correct my poor translation if needed :)

| Code                    | English                   | Chinese  |
|-------------------------|---------------------------|----------|
| encodingAndDecoding     | Encoding And Decoding     | 字符编码 |
| generator               | Generator                 | 产生器   |
| hash                    | Hash                      | 杂凑     |
| encryptionAndDecryption | Encryption And Decryption | 加密算法 |
| conversion              | Conversion                | 数据处理 |
| disassembler            | Disassembler              | 反编译   |
| obfuscation             | Obfuscation               | 混淆     |
| complier                | Complier                  | 编译器   |
| utility                 | Utility                   | 调试工具 |
| inspection              | Inspection                | 校验     |

I also did some changes to some of the tools, here's the list:
| Tool name     | Before         | After    |
|---------------|----------------|----------|
| uuid          | 信息摘要       | 产生器   |
| messageDigest | 信息摘要       | 杂凑     |
| ipAttribution | IP2Geo         | 数据处理 |
| crc           | 信息摘要, 校验 | 校验     |

# Tests

`node version: v12.22.1`

- [x] On-prem run `yarn dev`
- [x] On-prem run `yarn build`
- [x] Functionality of Changes 1
- [x] Functionality of Changes 2
- [x] Functionality of Changes 3 

Screen Shot 2022-05-18 at 00 02 42

Screen Shot 2022-05-18 at 00 02 49

Screen Shot 2022-05-18 at 00 02 54

Screen Shot 2022-05-18 at 00 03 11

Screen Shot 2022-05-18 at 00 02 31

Thanks again for the amazing tool :)

### CODE REVIEW

File: lang/en-US.js [hunk 1]
```diff
@@ -208,5 +208,17 @@ export default {
...
       desc: 'Convert temperature into multiple units'
     },
+  },
+  tags: {
+    encodingAndDecoding: 'Encoding And Decoding',
+    generator: 'Generator',
+    hash: 'Hash',
+    encryptionAndDecryption: 'Encryption And Decryption',
+    conversion: 'Conversion',
+    disassembler: 'Disassembler',
```
Comment (HoshinoSuzumi) [inline lang/en-US.js:218]: “反编译” 而不是 “反汇编”
'Decompiler' instead of 'Disassembler'

**Review by HoshinoSuzumi — CHANGES_REQUESTED** [review]
Review has been completed, please check.
Thank you very much for your contribution and support 😉

File: lang/zh-CN.js [hunk 2]
```diff
@@ -208,5 +208,17 @@ export default {
...
+    encodingAndDecoding: '字符编码',
+    generator: '产生器',
+    hash: '杂凑',
+    encryptionAndDecryption: '加密算法',
+    conversion: '数据处理',
+    disassembler: '反编译',
+    obfuscation: '混淆',
+    complier: '编译器',
+    utility: '调试工具',
+    inspection: '校验'
```
[the remark this answers is not in the document]
Comment (dextermallo) [reply lang/zh-CN.js]: Fixed in [fb9932b](https://github.com/UniiemStudio/CTFever/pull/48/commits/fb9932b734414725da095b649d31028721df93dd)

File: lang/zh-CN.js [hunk 3]
```diff
@@ -208,5 +208,17 @@ export default {
...
       desc: '将温度转换为多种单位'
     },
+  },
+  tags: {
+    encodingAndDecoding: '字符编码',
+    generator: '产生器',
+    hash: '杂凑',
+    encryptionAndDecryption: '加密算法',
+    conversion: '数据处理',
+    disassembler: '反编译',
```
[the remark this answers is not in the document]
Comment (dextermallo) [reply lang/zh-CN.js]: Fixed in [fb9932b](https://github.com/UniiemStudio/CTFever/pull/48/commits/fb9932b734414725da095b649d31028721df93dd)

File: lang/en-US.js [hunk 4]
```diff
@@ -208,5 +208,17 @@ export default {
...
+    obfuscation: 'Obfuscation',
+    complier: 'Complier',
+    utility: 'Utility',
+    inspection: 'Inspection'
```
[the remark this answers is not in the document]
Comment (dextermallo) [reply lang/en-US.js]: Fixed in [fb9932b](https://github.com/UniiemStudio/CTFever/pull/48/commits/fb9932b734414725da095b649d31028721df93dd)

File: lang/en-US.js (hunk 1 above)
Comment (dextermallo) [reply lang/en-US.js]: Fixed in [fb9932b](https://github.com/UniiemStudio/CTFever/pull/48/commits/fb9932b734414725da095b649d31028721df93dd)

**Review by HoshinoSuzumi — APPROVED** [review]
Thanks a lot for your contribution again! Have a nice day

**Pull request merged**